enterprise/server/util/vfs_server/vfs_server.go

```diff
@@ -41,6 +41,45 @@ const (
 	_OFD_SETLKW = 38
 )
 
+var (
+	syscallStatusENOENT    = makeSyscallErrStatus(syscall.ENOENT)
```

Comment:
Are syscall error status objects created frequently, and thus this change reduces allocations and memory pressure?

Reply:
The `Lookup` FUSE API is called frequently and it's common for it to be called for things that do not exist.

Reply:
IIRC this shaved about 25 million allocations over the duration of a BB build